### PR TITLE
Sru/19.2.36 azure

### PR DESCRIPTION
**Pull request by blackboxsw**

### CODE REVIEW

File: sru-templates/manual/azure-sru [hunk 1]
```diff
@@ -39,56 +71,76 @@ for series in $SRU_SERIES; do
...
-    ssh "${sshopts[@]}" $IP -- dpkg-query --show cloud-init
-    ssh "${sshopts[@]}" $IP -- sudo cat /run/cloud-init/result.json
-    ssh "${sshopts[@]}" $IP -- grep Trace /var/log/cloud-init.log
-    ssh "${sshopts[@]}" $IP -- systemd-analyze
-    ssh "${sshopts[@]}" $IP -- systemd-analyze blame
-    ssh "${sshopts[@]}" $IP -- cloud-init analyze show
-    ssh "${sshopts[@]}" $IP -- cloud-init analyze blame
-    echo 'Networking config'
-    ssh "${sshopts[@]}" $IP -- cat $netcfg
+	ssh "${sshopts[@]}" $IP -- cloud-init status --wait --long
```
Comment (OddBloke) [inline sru-templates/manual/azure-sru]: There seems to be a mixture of tabs and spaces in this file now. Could we remain consistent (on 3 spaces) for this PR, and have a follow-up to change the formatting?

Comment (OddBloke) [reply sru-templates/manual/azure-sru]: (Looking above, I see that we're already internally inconsistent. So let me rephrase: let's not change the indentation on existing lines in this PR, but instead change them _all_ to be consistent in a separate PR.)

File: sru-templates/manual/azure-sru [hunk 2]
```diff
@@ -39,56 +71,76 @@ for series in $SRU_SERIES; do
...
-    ssh "${sshopts[@]}" $IP -- dpkg-query --show cloud-init
-    ssh "${sshopts[@]}" $IP -- sudo cat /run/cloud-init/result.json
-    ssh "${sshopts[@]}" $IP -- grep Trace /var/log/cloud-init.log
-    ssh "${sshopts[@]}" $IP -- systemd-analyze
-    ssh "${sshopts[@]}" $IP -- systemd-analyze blame
-    ssh "${sshopts[@]}" $IP -- cloud-init analyze show
-    ssh "${sshopts[@]}" $IP -- cloud-init analyze blame
-    echo 'Networking config'
-    ssh "${sshopts[@]}" $IP -- cat $netcfg
+        ssh "${sshopts[@]}" $IP -- cloud-init status --wait --long
```
Comment (blackboxsw) [inline sru-templates/manual/azure-sru]: Because we are now nested inside a second for loop (to handle basic vs advanced networks) we have to indent all the scp/ssh operations within the additional for loop;

File: sru-templates/manual/azure-sru (hunk 1 above)
Comment (blackboxsw) [reply sru-templates/manual/azure-sru]: will separate and fix the manual sru template update spacing.
